src/Sarif/Schemata/Sarif.schema.json

```diff
@@ -86,6 +86,33 @@
       },
       "required": [ "locations" ]
     },
+    "exception": {
+      "type": "object",
+      "properties": {
+        "type": {
```

Comment:
This is the issue we're looking at in the other change PR I sent yesterday. I'd like to resolve it separately.

---
In reply to: [61636346](https://github.com/Microsoft/sarif-sdk/pull/229#discussion_r61636346) [](ancestors = 61636346)